examples/olm/install-resources.yaml

```diff
@@ -36,3 +30,9 @@ spec:
   source: security-profiles-operator
   #sourceNamespace: openshift-marketplace on OCP
   sourceNamespace: olm
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: security-profiles-operator
+  namespace: security-profiles-operator
```

Comment:
In case anyone is wondering, I just moved the OG to the end of the file so that it's easier to append to in the tests, as a byproduct we can keep using this example in the test which would hopefully also keep testing the example.